bookstack.json

```diff
@@ -1,36 +1,19 @@
 {
   "name": "bookstack",
   "plugin_schema": "2",
-  "release": "11.3-RELEASE",
-  "artifact": "https://github.com/Adirael/iocage-plugin-bookstack.git",
+  "artifact": "https://github.com/sam-harry/iocage-plugin-bookstack.git",
```

Comment:
I guess I'll need to keep this as `https://github.com/Adirael/iocage-plugin-bookstack.git` for everything to work properly once I merge the PR. Could you modify it? 

The rest looks great. Would you like to add a CONTRIBUTORS.md?

Thanks for the help!

Reply:
Done!

Having my name on the PR is enough for me, thanks though.